qa/rpc-tests/wallet_recover.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
```

Comment:
Add release notes saying that `-recoverwallet` only uses the emergency recovery phrase for now, not any other keys that are in the dump file (and that this will be fixed later).